backend/src/main/java/com/ez/ezbackend/budget/request/ScheduleRequest.java

```diff
@@ -0,0 +1,77 @@
+package com.ez.ezbackend.budget.request;
+
+import com.ez.ezbackend.budget.entity.Category;
+import com.ez.ezbackend.budget.entity.Schedule;
+import com.ez.ezbackend.budget.enums.RecurringPattern;
+import com.ez.ezbackend.shared.entity.User;
+import com.ez.ezbackend.shared.exception.EzIllegalRequestException;
+import com.google.common.base.Strings;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ScheduleRequest {
+  private Long categoryId;
+  private String description;
+  private BigDecimal deposit;
+  private BigDecimal withdraw;
+  private LocalDate startDate;
+  private RecurringPattern recurringPattern;
+
+  public static Schedule convertToSchedule(ScheduleRequest scheduleRequest) {
+    return convertToSchedule(scheduleRequest, null, null, null);
```

Comment:
I thought every schedule would belong to one user. Am I missing something here?

Reply:
Hmm.. we have the same thing in Transaction and Category as well...
I think it is probably better to have the validation in entity classes.